pkgs/by-name/mn/mni_autoreg/package.nix

```diff
@@ -10,13 +10,13 @@
 
 stdenv.mkDerivation {
   pname = "mni_autoreg";
-  version = "unstable-2022-05-20";
+  version = "0-unstable-2024-10-04";
```

Comment:
shouldn't this be 0.99.70-unstable-<...>?

Reply:
Yes, it should be.

Reply:
I learn something new every day, thank you guys!